adminSiteServer/apiRouter.ts

```diff
@@ -2318,10 +2322,12 @@ apiRouter.post("/posts/:postId/createGdoc", async (req: Request) => {
 
         const gdoc = new Gdoc(gdocId)
         gdoc.slug = post.slug
+        gdoc.tags = tags
+        gdoc.content.title = post.title
         gdoc.published = false
         gdoc.createdAt = new Date()
         gdoc.publishedAt = post.published_at
-        await dataSource.getRepository(Gdoc).insert(gdoc)
+        await dataSource.getRepository(Gdoc).save(gdoc)
```

Comment:
if I get this right, `insert` doesn't cascade the tags relationship, while `save` does. Maybe worth adding a note about this here?